src/ds3_uint64_string_map.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+#ifndef __DS3_UINT64_T_STRING_MAP__
+#define __DS3_UINT64_T_STRING_MAP__
+
+#include <stdlib.h>
+#include <glib.h>
```

Comment:
Since this file gets included by ds3.h we are leaking the fact that we're using glib.  This and other glib reference should be removed from this header.